src/main/java/com/conveyal/datatools/manager/controllers/api/FeedVersionController.java

```diff
@@ -262,6 +269,50 @@ private static FeedVersion publishToExternalResource (Request req, Response res)
         }
     }
 
+    /**
+     * HTTP controller that handles merging multiple feed versions for a given feed source, with version IDs specified
+     * in a comma-separated string in the feedVersionIds query parameter and merge type specified in mergeType query
+     * parameter. NOTE: REGIONAL merge type should only be handled through {@link ProjectController#mergeProjectFeeds(Request, Response)}.
+     */
+    private static String mergeFeedVersions(Request req, Response res) {
+        String[] versionIds = req.queryParams("feedVersionIds").split(",");
+        // Try to parse merge type (null or bad value throws IllegalArgumentException).
+        MergeFeedsType mergeType;
+        try {
+            mergeType = MergeFeedsType.valueOf(req.queryParams("mergeType"));
+            if (mergeType.equals(REGIONAL)) throw new IllegalArgumentException();
```

Comment:
Please populate the IllegalArgumentException with a string error.

Reply:
Hmm, I see that it's caught down below. Maybe it'd be good to specify that the REGIONAL type is not allowed for this endpoint.